praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
+
+        The additional parameters are passed directly into
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
+
+        """
+        url = self.config['default_subreddits']
+        if (len(args) < self.get_content.__code__.co_varnames.index('limit') and
+                'limit' not in kwargs):
```

Comment:
> But I don't think it would be confusing regardless.

I am pretty sure it will be confusing, especially for people who are new to praw/reddit api. Heck, I have quite a bit of exprience using praw, and after I implemented this method and tried to use it(without passing `limit=None`), and I was confused for a second until I remembered that default `limit` is just 25.

> you can add the limit=None caveat in the docstring via something like this

IMO, it would be much better to just have a sensible default. Unike other `get_content()`-based `get_default_subreddits()` will always return a short fixed list of items that rarely changes, so if you don't know about `limit` the perceived problem will be "wtf, why half of the default subreddits are missing" and not "oh, it just returned only 25 items out of that long list". 

Practicality beats purity.

> but, this is extremely ugly.

It is ugly, but it is not _that_ ugly. It is not like users of praw will have to deal with this code. The API stays clean. I could simply implement the function `def get_default_subreddits(self, limit=None, *args, **kwargs)`, which would look better, but then the order of arguments will be different from other `get_content()`-based generators.